styles.css

```diff
@@ -135,6 +136,53 @@ a#letter-icon {
   }
 }
 
+#nav-wrap {
+  height: 100vh;
+  width: 100vw;
+  background: rgb(60, 58, 57);
+  position: fixed;
+  top: 0;
+  left: 100%;
+  padding: 9%;
+  z-index: 11;
+  transition: 0.3s;
+}
+
+@media (min-width: 769px) {
+  #nav-wrap {
+    display: none;
+  }
+}
+
+#close-button {
+  position: absolute;
+  top: 1vh;
+  right: 2.5%;
+}
+
+#close-button img {
+  height: 50px;
+}
+
+#nav-m-wrap {
+  position: absolute;
+  top: 15%;
+  width: 82%;
+}
+
+#nav-m-wrap a {
+  width: 100%;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  padding: 10px;
+  border-bottom: 1px solid rgb(111, 108, 107);
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
 #first {
   /* display: none; */
```

Comment:
- [ ] Please remove this commented code, it's always a good practice to get rid of commented lines that will make a lot of noise in your code 💯 👍🏻 

- [ ] Always use the class attribute for multiple elements do not use the ID selector for multiple elements, Kindly change this 👍🏻 you can check the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 🤓👍🏻